<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opcja1</class>
 <widget class="QDialog" name="opcja1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="Powrot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Tablet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj Tablet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Stacjonarne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj Stacjonarne</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Przenosne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj Przenosne</string>
   </property>
  </widget>
  <widget class="QFrame" name="ramka">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Nazwa</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="textEdit_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Numer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTextEdit" name="textEdit_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Powrot</sender>
   <signal>clicked()</signal>
   <receiver>opcja1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Tablet</sender>
   <signal>clicked()</signal>
   <receiver>opcja1</receiver>
   <slot>dodajtablet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Stacjonarne</sender>
   <signal>clicked()</signal>
   <receiver>opcja1</receiver>
   <slot>dodajstacjonarne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Przenosne</sender>
   <signal>clicked()</signal>
   <receiver>opcja1</receiver>
   <slot>dodajprzenosne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>opcja1</receiver>
   <slot>dodaj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>powrot()</slot>
  <slot>dodajtablet()</slot>
  <slot>dodajprzenosne()</slot>
  <slot>dodajstacjonarne()</slot>
  <slot>dodaj()</slot>
 </slots>
</ui>
